.github/CONTRIBUTING.md

```diff
@@ -42,4 +42,4 @@ If there are any failures then the automated tests fail.
 These tests are run both with the latest available version of Nextflow and also the minimum required version that is stated in the pipeline code.
 
 ## Getting help
-For further information/help, please consult the [nf-core/openmspeptidequant documentation](https://github.com/nf-core/openmspeptidequant#documentation) and don't hesitate to get in touch on [Gitter](https://gitter.im/nf-core/Lobby)
+For further information/help, please consult the [nf-core/mhcquant documentation](https://github.com/nf-core/mhcquant#documentation) and don't hesitate to get in touch on [Gitter](https://gitter.im/nf-core/Lobby)
```

Comment:
Same question here, but maybe this has to be done in tools

Reply:
I added it here directly, but will open a separate issue for tools !